### ISSUE TITLE
Use thumbnails instead of buttons.

### ISSUE THREAD

**Issue by Anupam-dagar** [GeekArmy/r3public_of_gam3rs#4]
Change buttons to thumbnails for the game,it will look better.

**Comment by aswanthkoleri** [comment]
how about flipcards ?

**Comment by RavicharanN** [comment]
Yeah, flip cards with options for View-code, Play-game, Fork-repo

**Comment by Anupam-dagar** [comment]
Flipcards are kind of mainstream, jquery might have some cool effects, if we are not able to find a good one then it will be all good to use flip cards.
